generator/src/main/java/com/linkedin/pegasus/generator/TemplateSpecGenerator.java

```diff
@@ -712,6 +712,7 @@ private ClassTemplateSpec generateFixed(FixedDataSchema schema)
 
   private TyperefTemplateSpec generateTyperef(TyperefDataSchema schema, TyperefDataSchema originalTyperefSchema)
   {
+    pushCurrentLocation(_schemaResolver.nameToDataSchemaLocations().get(schema.getFullName()));
```

Comment:
as we discussed offline, Please confirm if the inline typeref's dataSchemaLocation is also using enclosingSchema's location.

Reply:
Junchuan, I confirmed this in my new added test case